ui/component/viewers/videoViewer/view.jsx

```diff
@@ -535,6 +536,7 @@ function VideoViewer(props: Props) {
         activeLivestreamForChannel={activeLivestreamForChannel}
         defaultQuality={defaultQuality}
         doToast={doToast}
+        claim={claim}
```

Comment:
Please clean this up if no longer using. Must always be mindful of what we pass as props because that determines the rendering frequency.